### ISSUE TITLE
Non-technical GUI for accessing repos

### ISSUE THREAD

**Issue by erinys** [xetdata/xet-tools#48]
From Snehesh via Discord:

TLDR
Is there a GUI similar to github desktop to share the repo with collaborators from arts that might not be as comfortable with CLI as technical folks?

DETAILS
Hi I work with folks from music and arts often. Xethub has made it super easy for me to maintain and version my contents. However, to share with my collaborators from the arts, it's been a bit of a nightmare for me. I can share the web link but they are not able to bulk access folder etc. So I end up having to set up streams with remote support. And they need my help with any slight updates or changes. I was wondering if GUI client like github for desktop or even github for desktop can be repurposed for Xethub?

**Comment by erinys** [comment]
For the GUI/desktop app request, I'm curious - how large are the assets that you are sharing with your collaborators? We recently released a non-technical interface for solo work ([https://xethub.com/workspace](https://xethub.com/workspace)) and are considering incorporating an option to use a similar view for all repos.

**Comment by sneheshs** [comment]
Thank you @erinys!!!

I just checked out the workspace UI. Looks interesting but I don't think I fully understood the use case. 

I have a test repository where I uploaded a single recording session videos. In practice, there would be lots of videos and lab sensor data that different collaborators (eg. artist, studio personnels, editors, developers, ML engineers, and designers) need to access. Each type have different requirements on what they need to access and when. At the moment, the coordination is pretty manual with lots of emails and google drive sharing/ versioning. Developers and ML engineers are avid git users so they are fine. But for others, asking them to use git is a huge challenge.

I did experiment with GitHub desktop. It is good for novice developers and monitor versioning but it comes with a few challenges:
- Cannot browse latest files or branches files -- only displays changes
- Interprets xet repo as LTS and shows xet internal branches
- Preview not available for different data types

Xethub writable stream repo mentioned in #47 is perfect for our use case (assuming not buggy 😁) The only thing missing is an GUI.

**Comment by erinys** [comment]
@snehesh Right now, workspaces are intended as a personal scratch space where you can drag in files from your desktop and get easy time travel (through the UI) on how they have changed over time without having to interact with Git repos. <br><br>Workspaces are still backed by Git repos and we are thinking about how to incorporate this as an alternate repo-level view for non-technical users, though there are size limitations around workspaces since it's web-based. We've also done some investigation a XetHub Desktop application. I'd love to hear more on your thoughts and expectations when you get the chance!

**Comment by allisonwhilden** [comment]
We are tracking this effort internally with [PRO-54](https://linear.app/xethub/issue/PRO-54/design-accessible-repo-experience). Closing this a duplicate. Thanks!
